de_templates/test_config_Ingest_AzureSql_SourceName.yaml

```diff
@@ -0,0 +1,66 @@
+source_name: Example
+
+ado_variable_groups_nonprod:
+  - amido-stacks-de-pipeline-nonprod
+  - amido-stacks-infra-credentials-nonprod
+  - stacks-credentials-nonprod-kv
+
+ado_variable_groups_prod:
+  - amido-stacks-de-pipeline-prod
+  - amido-stacks-infra-credentials-prod
+  - stacks-credentials-prod-kv
+
+
+# Workload config
+
+window_start_default: 2020-01-01
+window_end_default: 2020-01-31
+
+# Ingest sources
+
+enabled: true
+
+ingest_entities:
+  - version: 1
+    display_name: SalesLT.Product
+    enabled: true
+    schema: SalesLT
+    table: Product
+    columns: "*"
+    load_type: full
+    delta_date_column: null
+    delta_upsert_key: null
+  - version: 1
+    display_name: SalesLT.SalesOrderHeader
+    enabled: true
+    schema: SalesLT
+    table: SalesOrderHeader
+    columns: "SalesOrderID, RevisionNumber, OrderDate, DueDate, ShipDate, Status, OnlineOrderFlag, SalesOrderNumber, PurchaseOrderNumber, AccountNumber, CustomerID, ShipToAddressID, BillToAddressID, ShipMethod, CreditCardApprovalCode, SubTotal, TaxAmt, Freight, TotalDue, Comment, rowguid, ModifiedDate"
+    load_type: delta
+    delta_date_column: ModifiedDate
+    delta_upsert_key: SalesOrderID
+  - version: 1
+    display_name: SalesLT.SalesOrderDetailID
+    enabled: true
+    schema: SalesLT
+    table: SalesOrderDetail
+    columns: "SalesOrderID, SalesOrderDetailID, OrderQty, ProductID, UnitPrice, UnitPriceDiscount, LineTotal, rowguid, ModifiedDate"
+    load_type: delta
+    delta_date_column: ModifiedDate
+    delta_upsert_key: "SalesOrderID, SalesOrderDetailID"
```

Comment:
I don't think the config needs to be part of the template personally...the template would generate the structure for the new pipeline, which can include a template config file, but I think the actual config values might as well just be added directly once this is generated.

Reply:
Done - I left only two parameters:

```
"data_source_name": "{{ pipeline_name }}",
"data_source_type": "{{ data_source_type }}",
```